share/spice/hearthstone/hearthstone.handlebars

```diff
@@ -0,0 +1,31 @@
+<div class="zci--hearthstone--ia">
+    
+    <div class="zci--hearthstone--left">
+        <img class="zci--hearthstone--card" src="{{image}}"/>
+    </div>
+    
+    <div class="zci--hearthstone--right">
+        <h5>{{name}}</h5>
+        <p>{{rarity}} {{type}} for {{hero}}</p>
+        <p>
+            <span>Cost: <b>{{cost}}</b> <image class="zci--hearthstone--icon" src="http://hydra-media.cursecdn.com/hearthstone.gamepedia.com/8/8b/Mana_icon.png"/></span>
+            
+            {{#if attack}}
+            <span> - Attack: <b>{{attack}}</b> <image class="zci--hearthstone--icon" src="http://hydra-media.cursecdn.com/hearthstone.gamepedia.com/f/fe/Attack_icon.png"/></span>
+            {{/if}}
+            
+            {{#if health}}
+            <span> - Health: <b>{{health}}</b> <image class="zci--hearthstone--icon" src="http://hydra-media.cursecdn.com/hearthstone.gamepedia.com/1/17/Health_icon.png"/></span>
+            {{/if}}
```

Comment:
You'll need to use the `imageProxy` helper for all images, see https://duck.co/duckduckhack/spice_handlebars_helpers#imageproxy